lib/ruby_lsp/listeners/document_symbol.rb

```diff
@@ -47,12 +47,15 @@ def initialize(response_builder, dispatcher)
 
       sig { params(node: Prism::ClassNode).void }
       def on_class_node_enter(node)
-        @response_builder << create_document_symbol(
+        symbol = create_document_symbol(
           name: node.constant_path.location.slice,
           kind: Constant::SymbolKind::CLASS,
           range_location: node.location,
           selection_range_location: node.constant_path.location,
         )
+        return unless symbol
```

Comment:
These extra nil checks are a little clunky... another option might be to set `name` to some placeholder like `<blank>`.

Reply:
Oh, I like that idea. Much nicer than all of these early returns and allows the developer to find attributes or aliases that are missing names.